### ISSUE TITLE
crash on latest version

### ISSUE THREAD

**Issue by Dalethium** [Lightning-64/Tide#50]
```
net.minecraft.util.crash.CrashException: Rendering screen
	at net.minecraft.client.render.GameRenderer.render(GameRenderer.java:954) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1219) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.16.9.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.List.stream()" because the return value of "com.li64.tide.data.rods.CustomBaitLoader.getBaitData()" is null
	at com.li64.tide.util.BaitUtils.isBait(BaitUtils.java:34) ~[Tide-fabric-1.20.1-1.4.0.jar:?]
	at com.li64.tide.TideFabricClient.lambda$onInitializeClient$0(TideFabricClient.java:31) ~[Tide-fabric-1.20.1-1.4.0.jar:?]
	at net.fabricmc.fabric.api.client.item.v1.ItemTooltipCallback.lambda$static$0(ItemTooltipCallback.java:34) ~[fabric-item-api-v1-2.1.28+1802ada577-3d71adeb05625762.jar:?]
	at net.minecraft.item.ItemStack.handler$gop001$fabric-item-api-v1$getTooltip(ItemStack.java:12611) ~[client-intermediary.jar:?]
	at net.minecraft.item.ItemStack.getTooltip(ItemStack.java:791) ~[client-intermediary.jar:?]
	at net.minecraft.client.gui.screen.Screen.getTooltipFromItem(Screen.java:225) ~[client-intermediary.jar:?]
	at net.minecraft.client.gui.screen.ingame.HandledScreen.getTooltipFromItem(HandledScreen.java:183) ~[client-intermediary.jar:?]
	at net.minecraft.client.gui.screen.ingame.HandledScreen.drawMouseoverTooltip(HandledScreen.java:178) ~[client-intermediary.jar:?]
	at net.minecraft.client.gui.screen.ingame.InventoryScreen.render(InventoryScreen.java:92) ~[client-intermediary.jar:?]
	at net.minecraft.client.gui.screen.Screen.renderWithTooltip(Screen.java:110) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.mixinextras$bridge$method_47413$329(GameRenderer.java) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.wrapOperation$hik000$fancymenu$wrapRenderScreenFancyMenu(GameRenderer.java:12107) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.mixinextras$bridge$wrapOperation$hik000$fancymenu$wrapRenderScreenFancyMenu$330(GameRenderer.java) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.wrapOperation$kgf000$konkrete$wrapRenderScreenKonkrete(GameRenderer.java:17628) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.render(GameRenderer.java:945) ~[client-intermediary.jar:?]
	... 9 more
```

fabric 1.20.1 mod version 1.4.0, please let me know if you need more info!

**Comment by fadedwisp** [comment]
Same Here. i launch the server then when i load in and click on my hotbar my game crashes

**Comment by DatBoiOmonie** [comment]
same with me.

**Comment by Lightning-64** [comment]
Does this happen only when running a multiplayer server?

**Comment by Lightning-64** [comment]
Same issue as https://github.com/Lightning-64/Tide/issues/49, but on fabric. I'll put out a fix for this crash as soon as I can! Sorry about this.

**Comment by Lightning-64** [comment]
Fixed as of `Tide 1.4.0 [Fix]` :)
